src/Dapr.Client.Grpc/Protos/dapr.proto

```diff
@@ -1,106 +1,109 @@
-syntax = "proto3";
```

Comment:
Could you please update the gRPC sample code with this change with this PR as well so that they are not broken?

Reply:
> Could you please update the gRPC sample code with this change with this PR as well so that they are not broken?

Sure.

Reply:
> Could you please update the gRPC sample code with this change with this PR as well so that they are not broken?

Thanks Aman for pointing this out ! . I have now updated the sample and tested it.